crates/utils/re_video/src/decode/ffmpeg_h264/ffmpeg.rs

```diff
@@ -95,6 +93,14 @@ impl From<Error> for crate::decode::Error {
     }
 }
 
+/// In Annex-B before every NAL unit is a NAL start code.
+///
+/// Can also be 3 bytes of 0x00.
```

Comment:
Is this really correct? Isn't it `[0, 0, 1]`?

How does a parse know the difference between `[0, 0, 0] + [data starting on 1]` and `[0, 0, 0, 1] + [data]`?

Reply:
> Is this really correct? Isn't it [0, 0, 1]?

oh you're right, this was already wrong in the previous file.